src/smc-webapp/video-chat.cjsx

```diff
@@ -178,10 +178,10 @@ exports.VideoChatButton = rclass
 
     render: ->
         num_users_chatting = @video_chat.num_users_chatting()
+        style = {height : '31.4px'}
         if num_users_chatting > 0
-            style = {color: '#c9302c'}
-        else
-            style = {}
+            style.color = '#c9302c'
+
         <Button onClick={@click_video_button} style={style}>
```

Comment:
I'm not sure. I think it's not definitively the best way to do it but the rest of the spacing for file tabs is sort of written this way with very specific values.